AWSClientRuntime/Sources/Auth/AWSCredentialsProviderStaticConfig.swift

```diff
@@ -16,44 +16,15 @@ public struct AWSCredentialsProviderStaticConfig {
     
     public let sessionToken: String?
     
-    public init(accessKey: String,
-                secret: String,
-                sessionToken: String? = nil,
-                shutDownCallback: ShutDownCallback? = nil) {
+    public init(
+        accessKey: String,
+        secret: String,
+        sessionToken: String? = nil,
+        shutDownCallback: ShutDownCallback? = nil
+    ) {
         self.accessKey = accessKey
         self.secret = secret
         self.sessionToken = sessionToken
         self.shutDownCallback = shutDownCallback
     }
 }
-
-extension AWSCredentialsProviderStaticConfig {
-    func toCRTType() -> CredentialsProviderStaticConfig {
-        return CredentialsProviderStaticConfig(accessKey: accessKey,
-                                               secret: secret,
-                                               sessionToken: sessionToken,
-                                               shutDownOptions: shutDownCallback)
-    }
-}
-
-struct CredentialsProviderStaticConfig: CRTCredentialsProviderStaticConfigOptions {
```

Comment:
not required?

Reply:
Nope. This struct no longer has any purpose. 
Checkout the related factory method in [AwsCredentialsProvider]](https://github.com/awslabs/aws-sdk-swift/pull/777/files#diff-c858c06a42a458b91bb72c1ce5190f515a40cf4463775e74d8f8f91d4e53c506) to see why its no needed.